realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java

```diff
@@ -630,14 +633,15 @@ public void migration_backlinkedSourceFieldDoesntExist() {
      * for `BacklinksWrongTypeTarget` should notice, though, that its `parents` field points to an object
      * of the wrong type, `Integer`, instead of `BacklinksWrongTypeSource`.
      */
+    @Ignore("Need to rebuild the test library")
```

Comment:
After thinking about that, I am pretty sure that it will not work.  I need two different versions of exactly the same class.  Unless I'm missing something, there is no way to do that in a single compilation unit.